README.md

```diff
@@ -60,7 +60,7 @@ Once the installation is done, the next thing that you are going to do is to con
 
 #### Enable payment methods
 
-> Note that Alipay and Internet Banking payment methods are only available for merchants with a Thai-registered Omise account.
+> Note that Alipay, Bill Payment: Tesco, Installment, and Internet Banking payment methods are only available for merchants with a Thai-registered Omise account.
```

Comment:
Can we swap those two options were just added?

Reply:
what do you mean by swap?

Reply:
I mean put the Installment in front of the Bill Payment

Reply:
Never mind this comment. I just realized how you sort this list

Reply:
👍